elide-graphql/src/main/java/com/yahoo/elide/graphql/GraphQLConversionUtils.java

```diff
@@ -102,10 +102,10 @@ public GraphQLScalarType classToScalarType(Class<?> clazz) {
             return Scalars.GraphQLString;
         } else if (clazz.equals(BigDecimal.class)) {
             return Scalars.GraphQLBigDecimal;
-        } else if (Date.class.isAssignableFrom(clazz)) {
-            return GraphQLScalars.GRAPHQL_DATE_TYPE;
         } else if (scalarMap.containsKey(clazz)) {
             return scalarMap.get(clazz);
+        } else if (Date.class.isAssignableFrom(clazz)) {
```

Comment:
Why the reorder here?

Reply:
so that it calls newly added custom Date converters o/w as these new time grains extends Date so it was not getting called.